linux/Platform.c

```diff
@@ -662,6 +673,91 @@ bool Platform_getNetworkIO(NetworkIOData* data) {
    return true;
 }
 
+void Plateform_getBacktraces(Vector* frames, const Process* process, char **error) {
+   Vector_prune(frames);
```

Comment:
I put `Vector_prune`, because the fonction doesn't know what's inside the vector and I preferred to remove all frames for every case.  It would make more sense if I would have used `Vector_set` instead of `Vector_add`. I'll remove it.